utils/playbook.py

```diff
@@ -1,7 +1,7 @@
 import os
 from ansible_runner import run_async
 
-def run_playbook(provider: str, username: str, project_name: str, curr_working_dir: str, playbook: str, stage: str, extra_vars: dict = None):
+def run_playbook(provider: str, username: str, project_name: str, curr_working_dir: str, playbook: str, stage: str, extra_vars: dict ):
```

Comment:
extra_vars should be initiated as None, it is not a parameter that's used always